source/install/self-managed-changelog.md

```diff
@@ -61,7 +61,7 @@ Mattermost v6.5.0 contains medium severity level security fixes. [Upgrading](htt
  - The post menu is now only rendered on the root post on hover over.
  - Updated a library used for storing drafts and other data in browser storage.
  - Enabled performance telemetry tracking for production deployments not running in developer mode. This telemetry tracking is disabled when telemetry is toggled off.
- - Inactive server email notifications will now be sent to users occasionally if they haven't interacted with their server for 100 hours or more.
+ - Inactive server email notifications will now be sent to System Admins occasionally if they haven't interacted with their server for 100 hours or more. Inactivity is determined by reviewing all activity on the server. This feature can also be disabled using the ``Mattermost_FeatureFlags_EnableInactivityCheckJob`` feature flag.
```

Comment:
The only change I would recommend for 64 is changing from "... occasionally if they haven't interacted with their server..." to "... if there have been no telemetry events on their server..."

I'm not certain the feature flag can be disabled by the admin. @marianunez when you are back from vacation can you answer?